src/js/utils/Select.jsx

```diff
@@ -254,11 +261,18 @@ class Select extends Component {
       return null;
     };
 
+    const isTooltipEnabled = () => {
+      if (this.props.value.translatedName) {
+        return true;
+      }
+      return !(!showLabelTooltip || (showValueTooltip && this.props.value));
```

Comment:
this statement feels so complicated to me, I will try to think about a better statement and will let you know

Reply:
in my opinion to simplify it we could name the function as `isTooltipDisabled` since the `<Tooltip>`'s component property is named as `disabled`.
So we could have:
```
const isTooltipDisabled = () => {
    if (!this.props.value.translatedName) {
      return true;
    }
    return !showLabelTooltip || (showValueTooltip && this.props.value);
```

I know we should be avoiding negated boolean expression, but since the `<Tooltip>`'s component property is negated itself (it is `disabled` not `enabled`), I think the function should be named basing on Tooltip's component property.

It's up to code convention, so I'd like to hear from you @awalkowiak @jmiranda as we've had some discussions around it lately, what you think about it.
In my opinion leaving it as it is, the last statement is very hard to read